policies/travel-policy.md

```diff
@@ -0,0 +1,352 @@
+OpenSSL Travel Policy
+=====================
+
+Purpose
+-------
+
+When travelling for OpenSSL business related purposes, it is OpenSSL's
+intention to cover reasonable travel expenses.
+
+The purpose of this Travel Policy (The Policy) is to clarify the travel
+related expenses OpenSSL will reimburse, the level of reimbursement, the
+process for authorising travel and how to claim a reimbursement for
+approved travel expenses.
+
+Scope
+-----
+
+The Policy applies to all Committee Members, all persons under contract
+at OpenSSL, as well as other persons as designated by the OMC from time
+to time (subsequently known collectively as *Traveller*).
```

Comment:
(collectively probably implies plural)

```suggestion
to time (subsequently known collectively as *Travellers*).
```

Also, note that much of the document uses second-person "you", while some uses "traveller(s)", so not consistent.